transaction/core/src/ring_signature/rct_bulletproofs.rs

```diff
@@ -180,8 +183,8 @@ impl SignatureRctBulletproofs {
     pub fn verify<CSPRNG: RngCore + CryptoRng>(
         &self,
         message: &[u8; 32],
-        rings: &[Vec<(Address, Commitment)>],
-        output_commitments: &[Commitment],
+        rings: &[Vec<(CompressedRistrettoPublic, CompressedCommitment)>],
```

Comment:
This now takes compressed points, which can be read directly from the TxOuts in a transaction.